test/Grpc.IntegrationTests/OrchestrationPatterns.cs

```diff
@@ -23,10 +20,11 @@ public OrchestrationPatterns(ITestOutputHelper output, GrpcSidecarFixture sideca
     public async Task EmptyOrchestration()
     {
         TaskName orchestratorName = nameof(EmptyOrchestration);
-        await using DurableTaskGrpcWorker server = this.CreateWorkerBuilder()
-            .AddTasks(tasks => tasks.AddOrchestrator(orchestratorName, ctx => Task.FromResult<object?>(null)))
-            .Build();
-        await server.StartAsync(this.TimeoutToken);
+        await using AsyncDisposable server = await this.StartWorkerAsync(
+            b =>
```

Comment:
nit: seems like you could reduce a lot of extra whitespace in this file and other test files by moving the `b =>` onto the same line as the method call, i.e.:
```csharp
await using AsyncDisposable server = await this.StartWorkerAsync(b =>
{
    b.AddTasks(tasks => tasks.AddOrchestrator(orchestratorName, ctx => Task.FromResult<object?>(null)));
});
```

Is the current formatting a StyleCop thing or just your own personal preference?

Reply:
I think it was just I originally had two params to this function - in which case that is my preference for structuring it. I applied your suggestion since I removed the 2nd param.